+jrclust/+views/plotAuxCorr.m

```diff
@@ -107,12 +107,33 @@
             auxRate = hCfg.getOr('auxRate', hCfg.sampleRate);
         case {'.dat', '.bin'}
             if isempty(hCfg.auxChan)
-                return;
+                try % ask where it is 
+                    hCfg.auxFile = fullfile(hCfg.outputDir,uigetfile({'*.mat;*.csv'},...
+                    'Select the Aux file',hCfg.outputDir));
+                catch
+                    return;
+                end
             end
-
-            hRec = jrclust.detect.newRecording(hCfg.auxFile, hCfg);
-            auxSamples = single(hRec.readRawROI(hCfg.auxChan, 1:hRec.nSamples))*hCfg.bitScaling*hCfg.auxScale;
-            auxRate = hCfg.getOr('auxRate', hCfg.sampleRate);
+            try
+                hRec = jrclust.detect.newRecording(hCfg.auxFile, hCfg);
+                auxSamples = single(hRec.readRawROI(hCfg.auxChan, 1:hRec.nSamples))*hCfg.bitScaling*hCfg.auxScale;
+                auxRate = hCfg.getOr('auxRate', hCfg.sampleRate);
+            catch % data not from SpikeGLX
+                auxData = load(hCfg.auxFile);
```

Comment:
What format is this expecting?

Reply:
It works with `.mat` files (I wanted to add `.csv` as well but didn't work on it). This goes with the lines above:
```
if isempty(hCfg.auxChan)
                try % ask where it is 
                    hCfg.auxFile = fullfile(hCfg.outputDir,uigetfile({'*.mat;*.csv'},...
                    'Select the Aux file',hCfg.outputDir));
                catch
                    return;
                end
end
```  
This bit of code gives the opportunity to load an Aux file to people who don't use SpikeGLX . It worked, but didn't turn out to be as useful as I expected, so I haven't used it much.